vars/withElasticsearchClusterEnv.txt

```diff
@@ -0,0 +1,11 @@
+  Wrap the Elasticsearch cluster credentials and entrypoints as environment variables that are masked
```

Comment:
Not a blocker, but I'd prefer using the word `deployment` instead of `cluster` to follow the same terminology than Elastic Cloud

Reply:
for backward compatibility I'll keep `withClusterEnv` but change `withElasticsearchClusterEnv` for `withElasticsearchDeploymentEnv`, similar for `Kibana` and `Fleet`, is that ok?

Reply:
Perfect!